pkgs/development/python-modules/pydeprecate/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pyDeprecate";
```

Comment:
pname should stay the same as the attribute name, so without uppercase.
In fetchPypi you can use the one with uppercase

Reply:
thanks for the suggestion, fixed now